en/identity-server/7.0.0/docs/apis/restapis/user-organization.yaml

```
@@ -0,0 +1,323 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: 'WSO2 Identity Server - User Organization Management API Definition'
+  description: | 
+    This document specifies an **User Organization Management RESTful API** for **WSO2 Identity Server**.
+    
+    This API provides the capability to retrieve the resident organization of the authenticated user.
+  contact:
+    name: WSO2
+    url: 'http://wso2.com/products/identity-server/'
+    email: iam-dev@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://localhost:9443/t/{tenant-domain}/api/users/v1/me/organizations'
+    variables:
+      organization-domain:
+        default: 10084a8d-113f-4211-a0d5-efe36b082211
+security:
+  - OAuth2: []
+  - BasicAuth: []
+
+tags:
+  - name: me
+    description: Operations for the authenticated user
+
+paths:
+  /:
+    get:
+      tags:
+        - me
+      description:
+        Retrieve authorized organizations which matches the defined search criteria, if any.
+      summary:
+        This API is used to search and retrieve child organizations which are authorized for the user.
```

Seems summary and description is in reverse order.

```suggestion
      summary:
        Retrieve authorized organizations which matches the defined search criteria, if any.
      description:
        This API is used to search and retrieve child organizations which are authorized for the user.
```

---

This is now updated.